gomint-server/src/main/java/io/gomint/server/entity/EntityPlayer.java

```diff
@@ -82,12 +82,7 @@
 import io.gomint.server.world.WorldAdapter;
 import io.gomint.server.world.block.Block;
 import io.gomint.taglib.NBTTagCompound;
-import io.gomint.world.Chunk;
-import io.gomint.world.Gamemode;
-import io.gomint.world.Particle;
-import io.gomint.world.ParticleData;
-import io.gomint.world.Sound;
-import io.gomint.world.SoundData;
+import io.gomint.world.*;
```

Comment:
I don't think gomint code style currently uses star imports.

Reply:
Well I'm currently refactoring and I'm using GoMint's IntelliJ code style. Everything that exceeds 5/6 imports of the same imports IntelliJ replaces it with a star.